reg/drone/physics/electricity/Source.0.1.uavcan

```diff
@@ -0,0 +1,27 @@
+# A generic source or sink of electric power (battery, turbogenerator, braking resistor, etc.).
+# Low-pass filtering should be applied to avoid aliasing effects (as is the case everywhere else).
```

Comment:
Do we require to refer this on the data type description? Shouldn't this be at the spec level?

Reply:
I think it's sensible to provide it at the data type definition layer because it defines a usage contract. Somebody developing a publisher will be reminded that the data shall be low-pass filtered here.